src/app/containers/InlineLink/index.jsx

```diff
@@ -32,13 +32,23 @@ const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
     );
   }
 
-  // else return a normal hyperlink
+  if (isExternal) {
+    return (
+      <InlineLink
+        href={locator}
+        aria-label={`${blocks[0].model.text}${externalLinkText}`}
```

Comment:
```suggestion
        aria-label={`${blocks[0].model.text}${externalLinkText}`}
```

this line isn't really ideal but i'm not sure what I should be doing here

Reply:
At the very least I'd suggest constructing it before the return and then just referencing it here.